src/app/teacher/tags-filter/tags-filter.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatDividerModule } from '@angular/material/divider';
+import { SharedModule } from '../../modules/shared/shared.module';
+import { Tag } from '../../domain/tag';
+import { AbstractTagsMenuComponent } from '../abstract-tags-menu/abstract-tags-menu.component';
+
+@Component({
+  selector: 'tags-filter',
+  templateUrl: './tags-filter.component.html',
+  styleUrls: ['./tags-filter.component.scss'],
+  standalone: true,
+  imports: [CommonModule, MatDividerModule, SharedModule]
+})
+export class TagsFilterComponent extends AbstractTagsMenuComponent {
```

Comment:
Should this component be called SelectTagsComponent instead? 

TagsFilterComponent => SelectTagsComponent
tagsFilterChanged => selectedTagsEvent